### PR TITLE
feat: add gtmf integration VEKA

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -438,83 +436,81 @@ jobs:
...
+    strategy:
+      matrix:
+        services: [
+          'integrationveka'
+        ]
+    steps:
+    - name: Deploy services
+      env:
+        BUILD_URL: ${{ secrets.VBR_AWS_BUILD_API_DEVOPS }}/${{matrix.services}}
+        STATUS_URL: ${{ secrets.VBR_AWS_BUILD_STATUS_API_DEVOPS }}/${{matrix.services}}
```
Comment (ArneD) [inline .github/workflows/release.yml:475]: `vars.VBR_xxx`

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -438,83 +436,81 @@ jobs:
...
+      uses: informatievlaanderen/awscurl-polling-action/polling-action@main
+      with:
+          environment: tst
+          version: ${{ needs.release.outputs.version }}
+          status-url: $STATUS_URL
+          deploy-url: $BUILD_URL
+          access-key: ${{ secrets.VBR_AWS_ACCESS_KEY_ID_DEVOPS }}
+          secret-key: ${{ secrets.VBR_AWS_SECRET_ACCESS_KEY_DEVOPS }}
+          region: eu-west-1
+          interval: 2
```
Comment (ArneD) [inline .github/workflows/release.yml:485]: deploy-target: 'ecs_scheduled_task'

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -438,83 +436,81 @@ jobs:
...
+    needs: [ deploy_to_newtest_start_slack, release ]
+    name: Deploy to new test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        services: [
+          'integrationveka'
+        ]
+    steps:
+    - name: Deploy services
```
Comment (ArneD) [inline .github/workflows/release.yml:472]: deploy tasks

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -438,83 +436,81 @@ jobs:
...
+          environment: tst
+          version: ${{ needs.release.outputs.version }}
+          status-url: $STATUS_URL
+          deploy-url: $BUILD_URL
+          access-key: ${{ secrets.VBR_AWS_ACCESS_KEY_ID_DEVOPS }}
+          secret-key: ${{ secrets.VBR_AWS_SECRET_ACCESS_KEY_DEVOPS }}
+          region: eu-west-1
+          interval: 2
+
+    - name: Deploy services output
```
Comment (ArneD) [inline .github/workflows/release.yml:487]: Deploy tasks output

File: .github/workflows/release.yml (hunk 2 above)
Comment (jvandaal) [reply .github/workflows/release.yml:485]: Done.

**Review by ArneD — APPROVED** [review]

**Review by ArneD — APPROVED** [review]

**Pull request merged**